thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiPresenter.java

```diff
@@ -386,6 +386,10 @@ public void setUiThreadExecutor(@Nullable final Executor uiThreadExecutor) {
         mUiThreadExecutor = uiThreadExecutor;
     }
 
+    public TiTestPresenter<V> test() {
```

Comment:
Thought about this method a lot, especially if it is extensible enough for the future. It is. When my presenter requires additional setup for testing one can override this method. That's great 🚢